apis/certificate/certificate.yaml

```diff
@@ -1,12 +1,14 @@
 openapi: 3.0.1
 info:
-  title: Certificate api
-  description: This api is used for digital certificate assertion.
+  title: Certificate Generate API
+  description: |- 
+        This api is used for digital certificate assertion.
+        - [Testcases](https://app.getpostman.com/run-collection/4d683e7b17b98026c267)
   version: "1.0"
 servers:
-- url: https://staging.open-sunbird.org/api/certs/v1
+- url: https://staging.open-sunbird.org/api/certs
```

Comment:
let the url be https://staging.open-sunbird.org/api
and add the complete paths after this for all the APIs
For eg., for cert generate, path will be `/cert/v1/certs/generate`

Reply:
done